withdraw/wallet.go

```diff
@@ -17,7 +20,7 @@ type Store struct {
 // NewStore creates new instance of Store
 func NewStore() *Store {
 	return &Store{
-		ks: keystore.NewKeyStore("./unsafe_withdraw_keystore", keystore.StandardScryptN, keystore.StandardScryptP),
+		ks: keystore.NewKeyStore(fmt.Sprintf("./%s", keystoreDir), keystore.StandardScryptN, keystore.StandardScryptP),
```

Comment:
I'm not sure what the fmt.Sprintf is used for here?
Also when dealing with path, prefer using stuff like filepath.Join(...) for concatenation.

Reply:
Done.